src/autoscaler/scalingengine/cmd/scalingengine/main.go

```diff
@@ -54,6 +50,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	helpers.SetupOpenTelemetry()
```

Comment:
Shall we instrument also `gorilla/mux` also everywhere (e.g. `eventgenerator` etc.)? 

Like it was done here https://github.com/cloudfoundry/app-autoscaler-release/blob/94d5607e420353d9a824ce92ab69bc931db4faea/src/autoscaler/scalingengine/server/server.go#L34

Reply:
ah, good suggestion, yes.

Reply:
done in the same commit